### PR TITLE
intermediate css

### PR DESCRIPTION
**Pull request by RomanusBett**
-css sizing
-box modelling
-absolute positioning

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,22 +4,67 @@
 <head>
   <link rel="stylesheet" href="css/styles.css">
   <link rel="icon" href="favicon.ico">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Merriweather:wght@300&family=Montserrat:wght@300&family=Sacramento&display=swap" rel="stylesheet">
```
Comment (kevinbett) [inline index.html]: Use Atom beautify to align your code for easier readability

File: index.html [hunk 2]
```diff
@@ -4,22 +4,67 @@
...
   <title>Romanus personal website</title>
 </head>
 
 <body>
+
   <div class=" top-container
   ">
+  <img class="top-cloud"src="images/cloud.png" alt="cloud-img">
+  <img class="circular"src="images/romi.jpg" alt="">
```
Comment (kevinbett) [inline index.html]: tab image class here

**Review by kevinbett — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (RomanusBett) [reply index.html]: this has been fixed

**Review by kevinbett — APPROVED** [review]
Looks good 👍🏻

**Pull request merged**